src/borg/archive.py

```diff
@@ -988,6 +988,10 @@ def chunk_processor(data):
                 return chunk_entry
 
         item.chunks = []
+        # borg recreate: if we rechunkify, we'll get a fundamentally different chunks list, thus
+        # we need to get rid of .chunks_healthy, as it might not correspond to .chunks any more.
+        if getattr(self, 'recreate_rechunkify', False) and 'chunks_healthy' in item:
```

Comment:
`getattr(self, 'recreate_rechunkify'` smells bad

Reply:
iirc, the attribute is not always there, only when a recreate is performed.